.github/workflows/readme.yaml

```
@@ -10,19 +10,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      # Readme.com doesn't accept OpenAPI 3.1 yet, but we don't use many OpenAPI
-      # 3.1 features, so fake a downgrade and hope for the best.
-      #
-      # We also use api.simplified.yml, which is missing detailed MergePatch
-      # types, but which avoids using `type: "null"`.
+      # According to Readme.com: "We currently support everything up through 3.1.0"
+      # But we still need api.simplified.yml for graphql, so this is still required:
       - name: Set fake OpenAPI 3.0.1 version
         run: |
           perl -pi.bak -e 's/^openapi: .*$/openapi: 3.0.1/' static/api.simplified.yml
           diff -u static/api.simplified.yml.bak static/api.simplified.yml || true
+      # Need to remove const from api.simplified.yml (3.1 feature) 
+      # AND api.yml until it's supported by readme (ticket filed)
       - name: Remove const (support ticket filed)
         run: |
           perl -pi.bak -e 's/^\s+const: .+$//;' static/api.simplified.yml
           diff -u static/api.simplified.yml.bak static/api.simplified.yml || true
+          perl -pi.bak -e 's/^\s+const: .+$//;' static/api.yml
+          diff -u static/api.yml.bak static/api.yml || true
```

```suggestion
          perl -p -e 's/^\s+const: .+$//;' static/api.yml
```

#iamnotaperlmonk

why use .bak here?